test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -51,6 +52,7 @@ def test_telemetry_instance_tag_success_cpu(cpu, ec2_client, ec2_instance, ec2_c
 @pytest.mark.processor("neuron")
 @pytest.mark.integration("telemetry")
 @pytest.mark.parametrize("ec2_instance_type", ["inf1.xlarge"], indirect=True)
+@pytest.mark.skip("Feature doesn't exist on Neuron DLCs")
```

Comment:
is this a placeholder if we are skipping the test for neuron?

Reply:
Yes, this is a placeholder for when it is added in to the framework source code.